app/helpers/media_helper.rb

```diff
@@ -1,13 +1,10 @@
 module MediaHelper
   def media_thumbnail(media_item)
     if media_item.thumbnail?
-      if media_item.caption.text.present?
-        alt_text = truncate(media_item.caption.text, length: 36)
-        return image_tag(media_item.item.thumb.url, alt: alt_text)
-      else
-        # Use auto-generated alt text
-        return image_tag(media_item.item.thumb.url)
-      end
+      caption_text_exists = media_item.caption && media_item.caption.text
+
+      alt_text = caption_text_exists ? truncate(media_item.caption.text, length: 36) : nil
+      image_tag(media_item.item.thumb.url, alt: alt_text)
```

Comment:
The alt text is getting set to nil here in some cases. However, Rails has some special magic where is takes the file name and turns it into a readable alt tag if not other alt text has been provided. This line doesn't let Rails do it's magic, if that makes sense.

Can you adjust this so that if no caption text exists or if the caption text is an empty string, `alt:` does not get set inside the `image_tag`? That way the alt text can be auto generated when needed.

Thank you!